content/en/docs/ops/diagnostic-tools/istioctl/index.md

```diff
@@ -89,7 +89,10 @@ $ brew install bash-completion
 
 Once the `bash-completion package` has been installed on your macOS system, add the following line to your `~/.bash_profile` file:
 
-`[[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"`
+{{< text plain >}}
```

Comment:
Should this be bash instead of plain?

Reply:
It's simply text to cut and paste into another file.